### PR TITLE
NEST-536: Update OC

### PR DESCRIPTION
**Pull request by sarahelsaig**
[NEST-536](https://lombiq.atlassian.net/browse/NEST-536)

[NEST-536]: https://lombiq.atlassian.net/browse/NEST-536?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Piedone — CHANGES_REQUESTED** [review]
OK apart from the build failing.

**Review by Piedone — APPROVED** [review]

**Review by Piedone — CHANGES_REQUESTED** [review]
The OC version needs to be updated here too. And don't forget the submodules.

File: src/Lombiq.OSOCE.Web/Views/Settings-Cache.Edit.cshtml
Comment (Piedone) [inline src/Lombiq.OSOCE.Web/Views/Settings-Cache.Edit.cshtml:1]: These shouldn't be needed due to https://github.com/OrchardCMS/OrchardCore/pull/16607 being merged and thus in `2.0.0-preview-18315`.

**Review by Piedone — CHANGES_REQUESTED** [review]

File: src/Lombiq.OSOCE.Web/Views/Settings-Cache.Edit.cshtml
Comment (sarahelsaig) [reply src/Lombiq.OSOCE.Web/Views/Settings-Cache.Edit.cshtml:1]: Ah, I forgot to delete them! More specifically, I forgot because they had no effect.

**Review by Piedone — APPROVED** [review]

File: src/Lombiq.OSOCE.Web/Views/Settings-Cache.Edit.cshtml
Comment (Piedone) [reply src/Lombiq.OSOCE.Web/Views/Settings-Cache.Edit.cshtml:1]: More specifically, you forgot that you forgot.

Comment (sarahelsaig) [reply src/Lombiq.OSOCE.Web/Views/Settings-Cache.Edit.cshtml:1]: https://github.com/user-attachments/assets/807c0542-e897-4efb-8918-081fa61e0649

Comment (Piedone) [reply src/Lombiq.OSOCE.Web/Views/Settings-Cache.Edit.cshtml:1]: This always cracks me up :D.

**Review by Piedone — APPROVED** [review]

**Pull request merged**